emop-maven-plugin/src/main/java/edu/cornell/CleanMojo.java

```diff
@@ -13,28 +14,33 @@ public class CleanMojo extends edu.illinois.starts.jdeps.CleanMojo {
     /**
      * Options for which artifacts to clean.
      */
-    public static enum CleanMode {
-        EVERYTHING,         // All artifacts
-        VIOLATION_COUNTS    // violation-counts only
+    public static enum Artifact {
+        STARTS,
+        WEAVED_BYTECODE,
+        VIOLATION_COUNTS
     }
 
     /**
      * Parameter to determine which artifacts to clean.
      */
-    @Parameter(property = "clean.mode", defaultValue = "EVERYTHING")
-    private CleanMode mode;
+    @Parameter(property = "clean.mode", defaultValue = "STARTS,WEAVED_BYTECODE,VIOLATION_COUNTS")
+    private Set<Artifact> mode;
```

Comment:
can you explain in the comments how the multiple default values work, especially as you have "breaks" in your switch statement